### PR TITLE
INT-34 Updated candidate modal to match the rest of the application, …

### PR DESCRIPTION
**Pull request by PH86**
…added icons to dashcard. Removed redundand dummy data

### CODE REVIEW

**Review by JackHewitt08 — APPROVED** [review]

File: src/components/DashCard/DashCard.tsx [hunk 1]
```diff
@@ -15,11 +16,15 @@ const Card = styled.article`
...
   title,
   content,
+  icon,
 }: IDashCard): React.ReactElement => {
   return (
     <Card className="dashcard-container">
       <h2 className="dashcard-title">{title}</h2>
-      <h1 className="dashcard-content">{content}</h1>
+      <h1 className="dashcard-content">
+        {icon}
```
Comment (hrjuggins) [inline src/components/DashCard/DashCard.tsx]: Here you should probably just be passing the icon needed then render the font awesome component inside here. 

<FontAwesomeIcon className="dashboard-icon" icon={props.icon} />

File: src/pages/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -22,12 +30,48 @@ export const Dashboard: React.FC = (): React.ReactElement => {
...
         />
-        <DashCard title="Current Vacancies" content="9" />
-        <DashCard title="Current Applicants" content="40" />
-        <DashCard title="Interviews Today" content="4" />
-        <DashCard title="Callbacks Required" content="10" />
+        <DashCard
+          title="Active Vacancies"
+          content="9"
+          icon={
+            <FontAwesomeIcon className="dashboard-icon" icon={faNewspaper} />
```
Comment (hrjuggins) [inline src/pages/Dashboard/Dashboard.tsx]: Same as above ^^ 

Here you should probably just be passing the icon needed

<DashCard title="Active Vacancies" content="9"  icon={faNewspaper} />

**Review by hrjuggins — CHANGES_REQUESTED** [review]
Looks good mate. One small change to make with the Dashboard, Dashcard stuff.

There's a bit of repetitive code at the moment where you're passing the whole <FontAwesomeIcon /> component. Move that to the Dashcard and pass just the icon instead

File: src/components/DashCard/DashCard.tsx (hunk 1 above)
Comment (PH86) [reply src/components/DashCard/DashCard.tsx]: updated in latest commit

**Review by hrjuggins — APPROVED** [review]

**Pull request merged**